src/scale.ts

```diff
@@ -262,6 +262,11 @@ export interface Scale {
    */
   zero?: boolean;
 
+  /**
+   * The color space for scale/
+   */
+  interpolate?: 'rgb'| 'lab' | 'hcl' | 'hsl' | 'hsl-long' | 'hcl-long';
```

Comment:
I did not see `rgb-basis` in the example here, https://github.com/vega/vega/blob/1debe3fb58b33fa87ba66da73316210083be4537/spec/legends-quantitative.vg.json
So I was not sure if Jeff put all of D3 color spaces here or not.